tests/spread/plugins/go/legacy-build-run/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Build and run a basic Go snap with no base
+
+environment:
+  SNAP_DIR: ../snaps/go-hello
+
+systems: [ubuntu-*]
```

Comment:
What are you filtering out with this?

Reply:
I am considering someone will add a base called fedora, we will add tests for it and not run this.